ocaml/tests/test_gpu_group.ml

```diff
@@ -12,9 +12,14 @@
  * GNU Lesser General Public License for more details.
  *)
 
-open OUnit
 open Test_vgpu_common
 
+let assert_true msg b = Alcotest.(check bool) msg true b
+let check_ref_map msg x y =
```

Comment:
I'd change this to `assert_map_empty`, because it's important that we're comparing to the empty list below, therefore the order doesn't matter and we can use `list` instead of `slist`